algorl/src/TD.py

```diff
@@ -87,7 +90,10 @@ def td_control(self, algo, plot_name:str):
                 reward = self.env.grid[next_state]
 
                 # Compute the pair state-actions
-                q_values_df = algo(q_values_df, state, action, reward, next_state, next_action)
+                if lambda_target == False:
```

Comment:
You can replace it with 
``` if lambda_target:```

Reply:
Would you like to code to run like this: 

if lambda_target:
        q_values_df, eligibility_df = algo(q_values_df, eligibility_df, state, action, reward, next_state, next_action)
else:
        q_values_df = algo(q_values_df, state, action, reward, next_state, next_action)